modules/thunder_media/config/schema/thunder_media.schema.yml

```diff
@@ -7,3 +7,17 @@ thunder_media.settings:
     enable_filefield_remove_button:
       type: boolean
       label: 'Enable remove button'
+
+image.effect.thunder_media_auto_aspect:
+  type: mapping
+  label: 'Auto Aspect'
+  mapping:
+    landscape:
+      type: string
+      label: 'Landscape image style'
+    portrait:
+      type: string
+      label: 'Portrait'
```

Comment:
`Portrait image style` as this is what it is.

Reply:
Another thought - is this translatable? I can conceive of a use case where the image style is adding copyright text. But maybe let's leave that can-of-worms alone.